src/Git/GitStatusListCtrl.cpp

```diff
@@ -4490,3 +4531,78 @@ ULONG CGitStatusListCtrl::GetGestureStatus(CPoint /*ptTouch*/)
 {
 	return 0;
 }
+
+void CGitStatusListCtrl::LoadChangelists()
+{
+	CString changelistFLP = g_Git.m_CurrentDir + L"\\.tgitchangelist";
+	if (!PathFileExists(changelistFLP))
+		return;
+
+	try
+	{
+		CStdioFile file(changelistFLP, CFile::typeText | CFile::modeRead | CFile::shareDenyWrite);
+		CString changelistName = GITSLC_IGNORECHANGELIST;

```

Comment:
We shouldn't use a localized name here.

Reply:
What name is localized?
GITSLC_IGNORECHANGELIST seems to be not localized

Reply:
oh, true. Sorry for the noise here.